documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -112,19 +202,90 @@ docker run -it --rm --name connect \
     -e CONNECT_VALUE_CONVERTER_APICURIO_REGISTRY_GLOBAL-ID=io.apicurio.registry.utils.serde.strategy.AutoRegisterIdStrategy \
     -p 8083:8083 debezium/connect-apicurio:{debezium-docker-label}
 ----
+endif::community[]
+ifdef::product[]
+.. Follow the steps in the link:{LinkCDCGettingStarted}#deploying-kafka-connect[example of deploying Kafka Connect] in {NameCDCGettingStarted}.
+.. Open the `examples/kafka-connect/kafka-connect-s2i-single-node-kafka.yaml` file that you used to deploy Kafka Connect.
++
+Before you can create the connector instance,
+you must first enable connector resources in the `KafkaConnectS2I` Custom Resource (CR).
+
+.. In the `metadata.annotations` section, enable Kafka Connect to use connector resources.
++
+.kafka-connect-s2i-single-node-kafka.yaml
+[source,yaml,options="nowrap"]
+----
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaConnectS2I
+metadata:
+  name: my-connect-cluster
+  annotations:
+    strimzi.io/use-connector-resources: "true"
+spec:
+   ...
+----
 
+.. Apply the updated `kafka-connect-s2i-single-node-kafka.yaml` file to update the `KafkaConnectS2I` CR:
++
+`oc apply -f kafka-connect-s2i-single-node-kafka.yaml`
+
+.. In the Kafka Connect CR that defines the connector, add the properties that are required by the Avro converter. The CR looks like this:
++
+.inventory-connector.yaml
+[source,yaml,options="nowrap"]
+----
+  apiVersion: kafka.strimzi.io/v1beta1
+  kind: KafkaConnector
+  metadata:
+    name: inventory-connector  
+    labels:
+      strimzi.io/cluster: my-connect-cluster
+  spec:
+    class: io.debezium.connector.mysql.MySqlConnector
+    tasksMax: 1  
+    config:  
+      database.hostname: mysql  
+      database.port: 3306
+      database.user: debezium
+      database.password: dbz
+      database.server.id: 184054  
+      database.server.name: dbserver1  
+      database.whitelist: inventory  
+      database.history.kafka.bootstrap.servers: my-cluster-kafka-bootstrap:9092  
+      database.history.kafka.topic: schema-changes.inventory  
+      key.converter: io.apicurio.registry.utils.converter.AvroConverter
+      key.converter.apicurio.registry.url: http://apicurio:8080/api
+      key.converter.apicurio.registry.global-id: io.apicurio.registry.utils.serde.strategy.AutoRegisterIdStrategy
+      value.converter: io.apicurio.registry.utils.converter.AvroConverter
+      value.converter.apicurio.registry.url: http://apicurio:8080/api
+      value.converter.apicurio.registry.global-id: io.apicurio.registry.utils.serde.strategy.AutoRegisterIdStrategy
```

Comment:
Ok, so that's good to show this connector-specific configuration here. Taking a step back, there's still some inconsistency overall. The situation is this:

* Avro can be enabled on the Kafka Connect worker level (as shown above via "Configure a Kafka Connect instance with the following property settings", and also then via the environment variables in the community variant)
* Avro can be enabled for a specific connector (as shown here for product, it's currently not shown for the community variant)

My general recommendation would be to _only_ show the second approach, enabling it for a specific connector, and not mention the worker-level approach, as it's less flexible.

Reply:
I don't know how to implement your recommendation - that is, I don't know how to change the doc. I'll commit the changes I made and then perhaps you can suggest the specific changes you want to see. I'll be happy to accept/commit your suggestions. This time around, I didn't Commit Suggestions because in the past Chris has recommended one commit with all the updates.

Reply:
Ok. Let me know once you're done from your side and I'll try and make that change myself.

Reply:
In general though, the idea is to solely show how the Avro converter is used at the connector level instead of showing how to enable it globally for the Kafka Connect worker node. That's already the case for the product variant. For upstream it'd mean to show the right curl POST command with the equivalent configuration.

Reply:
I removed all references to workers. The doc now talks only about configuring a Debezium connector, upstream and downstream.